src/Common/ExtVarNames.cs

```diff
@@ -12,5 +12,6 @@ public static class ExtVarNames
         public const string UserAppInsightsKeyEnv = "APPINSIGHTS_INSTRUMENTATIONKEY";
         public const string UserAppInsightsEnableEnv = "ApplicationInsightsAgent_EXTENSION_VERSION";
         public const string UserAppInsightsConnectionStringEnv = "APPLICATIONINSIGHTS_CONNECTION_STRING";
+        public const string PythonEnableGunicornDynamicWorkerEnvVarName = "PYTHON_ENABLE_GUNICORN_DYNAMIC_WORKER";
```

Comment:
I am not sure I like this name...how about `PYTHON_ENABLE_GUNICORN_MULTIWORKERS`?